### DISCUSSION TITLE
More options

### DISCUSSION THREAD

**Discussion by dav1312** [official-stockfish/Stockfish#4462] [Ideas]
- [ ] Optimism
- [ ] 2-fold
- [ ] MultiPV options (explore all moves equally, give priority to good moves, or output from standard analysis [free multipv])
- [ ] Draw shift

**Comment by dubslow** [comment]
I like making optimism runtime adjustable, would make it easier to make kibitzers

**Reply by dav1312** [reply]
Looks like it doesn't help make the evaluation symmetrical

**Reply by dubslow** [reply]
It does help, it's just not the only contributor.

**Comment by vondele** [comment]
I don't like to add more options for these things. Less options is easier to reason about, to maintain and to simplify.
